pyproject.toml

```diff
@@ -1,4 +1,4 @@
 [tool.black]
 line-length = 120
-target-version = ['py35']
+target-version = ['py38', 'py39', 'py310', 'py311']
```

Comment:
You can only use the specific versions defined here? or for those onwards?

Reply:
You can use any version, but this will check that if you use any deprecated python feature, it will complain. And seems that also for certain new "linting patterns", [according to the docs](https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#t-target-version)